package.json

```diff
@@ -92,10 +93,6 @@
     "schema": "./src/prisma/schema.prisma"
   },
   "resolutions": {
-    "@thirdweb-dev/auth/**/axios": ">=1.7.8",
-    "@thirdweb-dev/auth/**/web3-utils": ">=4.2.1",
-    "ethers-gcp-kms-signer/**/protobufjs": ">=7.2.5",
-    "fastify/**/find-my-way": ">=8.2.2",
```

Comment:
These are needed to pin versions that don't have reported vulnerabilities. `synk test` on this branch brings back these vulns if we remove this:

```
✗ Cross-site Request Forgery (CSRF) [High Severity][https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459] in axios@0.27.2
✗ Regular Expression Denial of Service (ReDoS) [Medium Severity][https://security.snyk.io/vuln/SNYK-JS-AXIOS-6124857] in axios@0.27.2
✗ Cross-site Scripting (XSS) [Medium Severity][https://security.snyk.io/vuln/SNYK-JS-AXIOS-6671926] in axios@0.27.2
✗ Prototype Pollution [High Severity][https://security.snyk.io/vuln/SNYK-JS-WEB3UTILS-6229337] in web3-utils@1.10.4
```

Reply:
Some of these may be removable since I only see the first two deps in this vuln report list.